pkg/config/config.go

```diff
@@ -179,7 +179,7 @@ func (operatorConfig *NSXOperatorConfig) createTokenProvider() error {
 func (vcConfig *VCConfig) validate() error {
 	if len(vcConfig.VCEndPoint) == 0 {
 		err := errors.New("invalid field " + "VcEndPoint")
-		log.Info("validate VcConfig failed", "VcEndPoint", vcConfig.VCEndPoint)
+		log.V(2).Info("validate VcConfig failed", "VcEndPoint", vcConfig.VCEndPoint)
```

Comment:
it should be a Info level, since it's only reported once

Reply:
ok